### DISCUSSION TITLE
服务器端能不能屏蔽回国流量

### DISCUSSION THREAD

**Discussion by xcf13363175** [apernet/hysteria#543] [Ideas]
服务器端能不能屏蔽回国流量

**Comment by haruue** [comment]
服务端配置 acl 规则 `block country cn` 即可。

编写 acl 规则的文档见 https://hysteria.network/zh/docs/acl/ 。
编写完 acl 之后在服务端配置文件里加个 `"acl": "ACL文件名",` 来启用 acl ， 参考 [高级用法](https://hysteria.network/zh/docs/advanced-usage/) 。

**Reply by xcf13363175** [reply]
感谢回复
